__tests__/api/__utils__/seeds/users_and_follow_seeds.js

```diff
@@ -75,7 +80,8 @@ const generateUserAndFollowSeeds = async () => {
       allow_anon_dm: true,
       only_received_dm_from_user_following: false,
       is_backdoor_user: false,
-      verified_status: 'VERIFIED'
+      verified_status: 'VERIFIED',
+      is_karma_unlocked: true
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change adds the `is_karma_unlocked` property to the user object being pushed into the `bulks` array within the loop. As with the previous hunk, this change seems to be consistent with the rest of the code and does not introduce any logical errors or security issues. Again, it's important to ensure that this new property is handled correctly in other parts of the application where user data is processed.
```diff
+     is_karma_unlocked: true
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->